data/json/items/gun/ups.json

```diff
@@ -146,14 +148,17 @@
     "valid_mod_locations": [
       [ "emitter", 1 ],
       [ "grip", 1 ],
+      [ "heatsink", 1 ],
       [ "lens", 1 ],
+      [ "radiator", 1 ],
       [ "rail", 1 ],
       [ "sights", 1 ],
       [ "sling", 1 ],
       [ "stock", 1 ],
       [ "stock accessory", 1 ],
       [ "underbarrel", 1 ]
     ],
+    "default_mods": [ "power_save_emitter" ],
```

Comment:
This gun is incredibly heat efficient already. Adding this mod basically disables overheat mechanics for it, I dont think such buff is warranted.

Reply:
Indeed, the A7 laser rifle is already a very efficient laser gun when compared to other laser guns.
Adding just one of these mods will increase the firepower by roughly 32% until overheating, so the opinion that it is "too much" is understandable.
However, it is doubtful that it is so effective that the mechanism becomes ineffective.
I have gone from 20 rounds to overheat to 34 rounds, +14 rounds, but this still causes me to overheat frequently, and each time I do, I have to wait 50 turns.
But since this is not the argument I want to make, I have no objection to the idea that these numbers should be smaller.